contracts/ERC721A.sol

```diff
@@ -60,21 +60,22 @@ contract ERC721A is Context, ERC165, IERC721, IERC721Metadata, IERC721Enumerable
     constructor(string memory name_, string memory symbol_) {
         _name = name_;
         _symbol = symbol_;
+        _nextTokenId = 1;
```

Comment:
Should not be hardcoded. Users should have the flexibility to choose if they want to start from 0 or 1, minimally.

I've written a version that uses a function override at https://gist.github.com/Vectorized/e6f47b0c1de008bb5983852224e41f69.